mitosheet/mitosheet/public/v3/sheet_functions/README.md

```diff
@@ -0,0 +1,28 @@
+# Guide to Writing Sheet Functions
+
+## Context on Formulas
+From frontend to backend (in order), a formula will have the following order of operations:
+1. The user edits a cell to include a formula. They'll see some information in a dropdown below the cell editor that's autogenerated from a docstring in the python function for that formula. If you edit that docstring or create a new function, you'll want to run `mitosheet/docs/make_function_docs.py update_frontend` to update the information displayed in that dropdown. 
+2. Validation is run on the frontend to check for things like the validity of the columns used as input and whether the formula exists.
+3. The parser takes the raw string that the user created, and converts it to python code. This is a good step for debugging! You can see what the parser is generating by looking at the code chunk, and you can see exactly what your python function is going to be getting as input.
+4. **Arguments to the formula function:** When you write a formula in the mitosheet, users can use three things as arguments: a column, a cell, or a constant value. As such, when you write a formula's arguments, you'll need to handle both the case that an argument is a series, and the case that the argument is a constant. There are helper functions to make this more manageable! Namely, the `get_series_from_primitive_or_series` function will make it easy to make sure you're exclusively dealing with series to simplify the code. 
```

Comment:
They can also be a range reference! E.g. if someone references `A1:A4` - in your case, you didn't really need to handle these. 

RollingRange is the name of the object the parser generates and passes in this case, in the `rolling_range.py` file.